app/src/main/resources/com/oracle/javafx/scenebuilder/app/preferences/Preferences.fxml

```diff
@@ -110,9 +110,12 @@
       </items>
     </ChoiceBox>
 
-    <Separator prefWidth="-1.0" GridPane.columnIndex="0" GridPane.columnSpan="2" GridPane.rowIndex="16" />
+    <Label text="%prefs.animate.accordion" GridPane.columnIndex="0" GridPane.rowIndex="16" />
```

Comment:
Maybe a new separator is required in between the number of recent items and the new accordion option, as these two are not related.